source/_integrations/tuya.markdown

```diff
@@ -25,79 +24,123 @@ ha_platforms:
-- **Scene** - The device state in frontend panel will not change immediately after you activate a scene.
-- **Switch** - The platform supports switch and socket.
+It applies to smart devices that you have added to the Tuya Smart or Smart Life app.
 
 {% include integrations/config_flow.md %}
 
-During configuration, be careful to select the [country code](https://www.countrycode.org/) and the platform corresponding to those used by you in the app. Once configuration flow is completed, the devices configured in your app will be automatically discovered. Additionally, if your username or e-mail address isn't accepted, please try using your phone number (minus the country code) as your username.
+## Supported device categories
+- **Climate**: Supports air conditioner, heater, thermostat.
+- **Cover**: Supports curtain, curtain switch.
+- **Fan**: Supports fan, air purifier, humidifier, dehumidifier.
+- **Light**: Supports light, light strip, ambient light, light string, humidifier's light, Ceiling light.
+- **Scene**: Supports smart scene.
+- **Switch**: Supports switch, socket, power strip, smart kettle, breaker, pet water feeder, air purifier, diffuser.
+
+## Quick installation and startup
+
+### Configure on the Tuya IoT Platform
+
+#### Prerequisites
+
+You have created an account for [Tuya Smart or Smart Life app](https://developer.tuya.com/en/docs/iot/tuya-smart-app-smart-life-app-advantages?id=K989rqa49rluq#title-1-Download).
+
+#### Create a project
+
+1. Log in to the [Tuya IoT Platform](https://iot.tuya.com/?_source=627739af7d7e42379954f5c26a1c1759).
+2. In the left navigation bar, click **Cloud** > **Projects**.
+3. On the page that appears, click **Create**.
+4. In the **Create Project** dialog box, configure **Project Name**, **Description**, **Industry**, and **Availability Zone**. In the **Development Method** field, select **Smart Home PaaS** from the dropdown list.
+	> **Note:** Select all available zones.
+	
+![](../images/integrations/tuya/image_001.png)
+	
+5. Click **Create** to continue project configuration.
+6. In **Configuration Wizard**, select **Device status notification** API.
+   > **Note:** **Smart Home PaaS** API has been selected as default.
+   
+![](../images/integrations/tuya/image_002.png)
+
+7. Click **Authorize**.
+
+#### Get authorization key
+
+Click the created project to enter the **Project Overview** page and get the **Authorization Key** used to make API calls.
+
+![](../images/integrations/tuya/image_003.png)
 
-## Integration Options
+#### Link devices by app account
 
-It is possible to change various behaviors through the integration options, some common for integration and others specific to each `light` and `climate` devices. These can be changed at **Tuya** -> **Options** on the Integrations page.
+Link devices by your app account and copy the **Device ID** in the **Device List** to the clipboard for specifying `device_id` later.
 
-### Common Options
+**1.** Navigate to the **Devices** tab.
```

Comment:
We don't need to add emphasis to all words. Please reduce the use of emphasis used.